### PR TITLE
Added design links

### PR DESCRIPTION
**Pull request by ExpressHermes**
Hello, I have a added portfolio sites of various developers as an insipiration to students building their own portfolio. 

Issue #13764

### CODE REVIEW

**Comment by ExpressHermes** [comment]
@I3uckwheat please review this PR.

File: getting_hired/project_portfolio.md [hunk 1]
```diff
@@ -14,6 +14,38 @@ Remember to write some documentation for both technical and non-technical audien
...
+
+  * [Stratis Bakas](https://stratisbakas.com/)
+  * [Matt Farley](https://mattfarley.ca/)
+  * [Dejan Markovic](https://www.dejan.works/)
+  * [Rafael Caferati](https://caferati.me/)
+  * [Ian Lunn](https://ianlunn.co.uk/)
+  * [Ben Adam](https://benadam.me/)
+  * [Seb Kay](https://sebkay.com/)
+  * [Jonny MacEachern](https://jonny.me/)
+  * [Jack Jeznach](jacekjeznach.com)
```
Comment (marvingay) [inline getting_hired/project_portfolio.md:29]: This link in the markdown needs to be fixed.

File: getting_hired/project_portfolio.md [hunk 2]
```diff
@@ -14,6 +14,38 @@ Remember to write some documentation for both technical and non-technical audien
 
 * [A Beginner’s Guide to Writing Documentation](http://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
 
+### Design Inspirations
```
Comment (marvingay) [inline getting_hired/project_portfolio.md:17]: A brief sentence or two to introduce what's in the drop-down belongs here.

File: getting_hired/project_portfolio.md [hunk 3]
```diff
@@ -14,6 +14,38 @@ Remember to write some documentation for both technical and non-technical audien
...
+<details markdown="block">
+  <summary> Links for designs </summary>
```
Comment (marvingay) [inline getting_hired/project_portfolio.md:19]: "Show Designs" in conjunction with the introduction is more appropriate.

File: getting_hired/project_portfolio.md (hunk 2 above)
Comment (ExpressHermes) [reply getting_hired/project_portfolio.md]: Added an intro line

File: getting_hired/project_portfolio.md (hunk 3 above)
Comment (ExpressHermes) [reply getting_hired/project_portfolio.md:21]: I have made requested changes.

File: getting_hired/project_portfolio.md (hunk 1 above)
Comment (ExpressHermes) [reply getting_hired/project_portfolio.md]: I have fixed this link.

**Review by marvingay — APPROVED** [review]

**Comment by marvingay** [comment]
@ExpressHermes  Thank you so much for your contribution!

**Pull request merged**